libavcodec/hevcdec.h

```diff
@@ -471,6 +471,10 @@ typedef struct HEVCContext {
     int enable_parallel_tiles;
     atomic_int wpp_err;
 
+    /* Allow tiles and WPP to be disabled.  Enabled by default */
```

Comment:
Adding this since for now SSIMWAVE's threading is not compatible with WPP.

FFmpeg will be unaffected; slice threading mode with tiles and wpp are still valid.

However in SSIMWAVE's threading approach of:
- configure in "frame" mode and then implement intra-frame threading in our own code base - our own intra-threading will only work with tiles, not WPP.  So we need to tell it to not try WPP.   There will be code in big that flips allow_parallel_wpp off.